test/check-application

```diff
@@ -1178,12 +1178,19 @@ class TestFiles(testlib.MachineCase):
         b.wait_visible("[data-item='file0'].row-selected")
 
         b.key("ArrowDown")
-        b.wait_js_cond("""ph_is_visible("[data-item='file2'].row-selected") ||
-                          ph_is_visible("[data-item='file3'].row-selected")""")
+        b.wait_visible("[data-item='file2'].row-selected")
 
         b.key("ArrowUp")
         b.wait_visible("[data-item='file0'].row-selected")
 
+        # Test with very long hostnames
+        original_hostname = m.execute('hostnamectl hostname').strip()
+        self.addCleanup(m.execute, ['hostnamectl', 'set-hostname', original_hostname])
+        # length of testing farm hostname
+        m.execute('hostnamectl set-hostname aaaaaaaa-bbbb-cccc-dddd-eeeeeeeeee.testing-farm')
```

Comment:
Please don't cement this -- as you don't want to, I'll file a PR to throw out the host name. It just takes away precious space, is pointless and redundant (we already show the host name in the shell), and isn't related to the file system at all. Can you please test this with a long directory name instead?

Reply:
Well this produces the exact issue we have in CI and I don't think we agreed on throwing out the hostname at all.